pkg/sdkclient/sideinput/client.go

```diff
@@ -19,24 +20,21 @@ type client struct {
 	grpcClt sideinputpb.SideInputClient
 }
 
-var _ Client = (*client)(nil)
-
-// New creates a new client object. SideInput doesn't require server info to start ATM.
-func New(inputOptions ...sdkclient.Option) (*client, error) {
+// New creates a new client object.
+func New(serverInfo *info.ServerInfo, inputOptions ...sdkclient.Option) (Client, error) {
```

Comment:
Please look into e2e testing failures. Might be because we haven't updated the test images.